packages/cw1/src/helpers.rs

```diff
@@ -42,7 +42,7 @@ pub struct Cw1CanonicalContract(pub CanonicalAddr);
 impl Cw1CanonicalContract {
```

Comment:
You can remove this entirely, as I we are moving away from CanonicalContracts.

Fine if you leave this for now to make the PR smaller, and I can take a second PR for that one.